admin/src/scenes/volontaires/components/ModalAffectation.jsx

```diff
@@ -527,13 +527,13 @@ const HitPdr = ({ hit, onSend, data, young }) => {
           <div className="text-gray-500">
             Départ :{" "}
             <span className=" text-gray-900">
-              {formatStringDateWithDayTimezoneUTC(data?.ligneBus.departuredDate)} {data?.ligneToPoint.departureHour}
+              {formatDateFR(data?.ligneBus.departuredDate)} {data?.ligneToPoint.departureHour}
```

Comment:
Pour la gestion des fuseau horaire il y a maintenant la fonction `getZonedDate` qu'on utilise dans le formatage des date de debut et de fin du cohort.
tu peux donc reprendre quelque chose comme ça :
```suggestion
              {formatDateFR(getZonedDate(data?.ligneBus.departuredDate))} {data?.ligneToPoint.departureHour}
```
à tester.